### ISSUE TITLE
WARNING:absl:Importing a function (__inference_sync_batch_normalization_13_layer_call_and_return_conditional_losses_20965) with ops with custom gradients. Will likely fail if a gradient is requested.

### ISSUE THREAD

**Issue by aynesss** [google-research/simclr#195]
TF2.4.   Here is what I write to begin finetuning:
```
target_model_path = "gs://simclr-checkpoints-tf2/simclrv2/pretrained/r50_2x_sk1/saved_model" 
saved_model = tf.keras.models.load_model(target_model_path)
saved_model.model.save_weights('./weights.h5')
simclr_model.load_weights('./weights.h5', by_name=True, skip_mismatch=True)
```
But I found many lines :+1:
```
WARNING:absl:Importing a function (__inference_sync_batch_normalization_57_layer_call_and_return_conditional_losses_47179) with ops with custom gradients. Will likely fail if a gradient is requested.
WARNING:absl:Importing a function (__inference_sync_batch_normalization_21_layer_call_and_return_conditional_losses_22737) with ops with custom gradients. Will likely fail if a gradient is requested.
WARNING:absl:Importing a function (__inference_sync_batch_normalization_21_layer_call_and_return_conditional_losses_22737) with ops with custom gradients. Will likely fail if a gradient is requested.
WARNING:absl:Importing a function (__inference_sync_batch_normalization_21_layer_call_and_return_conditional_losses_22737) with ops with custom gradients. Will likely fail if a gradient is requested.
WARNING:absl:Importing a function (__inference_sync_batch_normalization_13_layer_call_and_return_conditional_losses_20965) with ops with custom gradients. Will likely fail if a gradient is requested.
WARNING:absl:Importing a function (__inference_sync_batch_normalization_13_layer_call_and_return_conditional_losses_20965) with ops with custom gradients. Will likely fail if a gradient is requested.
WARNING:absl:Importing a function (__inference_sync_batch_normalization_13_layer_call_and_return_conditional_losses_20965) with ops with custom gradients. Will likely fail if a gradient is requested.

```
I can't understand the reason?
